.github/scripts/classify_workflow_errors.py

```
@@ -0,0 +1,283 @@
+
+COMMENT_MARKER = "<!-- ci-error-classifier -->"
+
+
+# ---------------------------------------------------------------------------
+# GitHub helpers (all via gh CLI)
+# ---------------------------------------------------------------------------
+
+
+def gh_api(path: str) -> Any:
+    """Call GitHub API via ``gh`` CLI.  Returns parsed JSON or *None*."""
+    r = subprocess.run(
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)